### DISCUSSION TITLE
Settings: 'Execute now' button: Ctrl/Cmd + click to temporarily direct output to the 'Ask after execution' modal

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#174] [Ideas - done & released]
`Ctrl`/`Cmd` + click on the _Execute now_ icon could ease up testing by defining that `stdout` and `stderr` outputs should be directed to the [Ask after execution modal](https://publish.obsidian.md/shellcommands/Output+handling/Output+channel+-+Ask+after+execution) regardless of what the shell command's normal output channels happen to be.

Then it would be easy to debug why a shell command does not give the correct output e.g. when normally using [Open a file](https://publish.obsidian.md/shellcommands/Output+handling/Output+channel+-+Open+a+file) output channel, by temporarily using the above-mentioned modal instead of _Open a file_.

**Comment by FelipeRearden** [comment]
Fantastic idea @Taitava !!!!!
